Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -342,14 +285,58 @@ WarpX::computePhiRZ (const amrex::Vector<std::unique_ptr<amrex::MultiFab> >& rho
         );
     }
 
+#ifndef AMREX_USE_EB
+
+    // Define the linear operator (Poisson operator)
+    MLNodeLaplacian linop( geom_scaled, boxArray(), DistributionMap() );
+
+#else
```

Comment:
Is there a reason not to use the `MLEBNodeFDLaplacian` (used in the EB case) for the non-EB case as well? In recent tests I've done, the MLMG solver fails to converge when using `MLNodeLaplacian` but works fine with the EB version.

Reply:
@RemiLehe Maybe you have an answer for this question?

Reply:
I think that this would be a question for @WeiqunZhang

Reply:
Discussed with @WeiqunZhang today: `MLNodeLaplacian` should indeed be replaced by `MLEBNodeFDLaplacian` here.

Reply:
@Yin-YinjianZhao Could you make the corresponding change?